src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,14 @@
 package core.basesyntax;
 
 public class MainApp {
-
+    public static void main(String[] args) {
+        Machine[] technic = new Machine[3];
+        technic[0] = new Truck();
+        technic[1] = new Bulldozer();
+        technic[2] = new Excavator();
```

Comment:
```suggestion
    Machine[] technic = {
       new Truck(),
       new Bulldozer(),
       new Excavator()
};
```
you can initialize it directly